_posts/2019-03-09-Proxy-post-3.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Lorem ipsum dolor sit amet, consectetur
```

Comment:
Any reason we need to merge in the testing post titles into master? I think it'd look better to keep the Lorem Ipsum etc as is.

Reply:
Makes sense. No problem.